packages/notification-config-process/test/api/updateTenantNotificationConfig.test.ts

```diff
@@ -86,7 +86,7 @@ describe("API POST /tenantNotificationConfigs test", () => {
 
   it.each([
     { body: {} },
-    { body: { agreementSuspendedUnsuspendedToProducer: "invalid" } },
+    { body: { enabled: "invalid" } },
```

Comment:
I had not adapted this to the new config, so it was testing a less useful error case